File: 1822_SignoftheProductofanArray/1822_SignoftheProductofanArray.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var arraySign = function (nums) {
  let num = 1;
  if (nums.includes(0)) return 0;
  for (let i = 0; i < nums.length; i++) {
    nums[i] > 0 ? num : (num *= -1);
  }
  return num;
};

File: 1822_SignoftheProductofanArray/1822_SignoftheProductofAnArray.py
def arraySign(nums):
    num = 1
    if 0 in nums:
            return 0
    for i in range(len(nums)):
          if nums[i] <0:
                num *= -1
    return num


print(arraySign([-1,3,1,2,3,4,6,-3,-2]))